From private repo

v-alje opened the pull request.

h0n commented:
@mimig1

azurecla commented:
Hi __@h0n__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (hawong). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

mimig1 commented:
#sign-off

barbkess committed:
populate replicated table cache

itechedit committed:
Update custom-decision-service-log-downloader-reference.md

nitinme committed:
acrolinx fixes

curtand committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into linkedin0306

curtand committed:
update pic

curtand committed:
update pic

mmccrory committed:
updated the doc

bwren committed:
Link fix

nitinme committed:
image updates

mmccrory committed:
updated the doc

nitinme committed:
image updates

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-databricks-tutorial-eventshub

anrothMSFT committed:
add link to api reference

JeffGoldner committed:
Add Dell EMC CloudLink

mmccrory committed:
updated the doc

BrentL-Collabera committed:
Update Saas Applications Technical Publishing Guide.md

Fixed all blocking issues.

linda33wj committed:
Update quickstart descriptions

rayne-wiselman committed:
new article, fixing issues 5757,5756,5738

rayne-wiselman committed:
Update hyper-v-prepare-on-premises-tutorial.md

rayne-wiselman committed:
Update vmware-azure-common-questions.md

carolinacmoravia committed:
M24377:  Possible incorrect term: "IoT Hub and Device Provisioning Service"

Hello, @JimacoMS2,

Translator has reported possible source content issue.  

Please, help to check my proposed file change into the article and help to merge if you agree with fix. If not, please, let me know either if you would like me to fix it in another way within this PR, if you prefer to fix it in another PR, or if I should close this PR as by-design. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR. 

Many thanks in advance.

carolinacmoravia committed:
M24393: Typo mistake: "on" is missing between "information" and "the machine"

Hello, @georgewallace, 

Translator has reported possible source content issue.  

Please, help to check my proposed file change into the article and help to merge if you agree with fix. If not, please, let me know either if you would like me to fix it in another way within this PR, if you prefer to fix it in another PR, or if I should close this PR as by-design. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR. 

Many thanks in advance.

rwike77 committed:
Adding overview article

mmccrory committed:
updated the doc

ecfan committed:
Fix typos and punctuation

curtand committed:
add back in lost changes from repo refresh

mattbriggs committed:
adding web api clarification

mattbriggs committed:
update Setting nav.

rwike77 committed:
updated content

CarlRabeler committed:
[sql db] 1 TB limits fixed

ecfan committed:
Fix typos and other tweaks

CarlRabeler committed:
more

PRMerger13 committed:
Merge pull request #35312 from mmacy/stg-sse-byok

[STG] SSE custom keys FAQ update

ggailey777 committed:
PaulBatum review comments

ktoliver committed:
Merge pull request #35214 from mmccrory/LowPriDoc

Added Low Priority Doc on VMSS

PRMerger14 committed:
Merge pull request #35423 from ecfan/mvc-logic-apps-vs

Fix typos and punctuation

dlepow committed:
HPC VMs: RDMA clarification

curtand committed:
[Azure AD admin roles] remove preview from Information Protection

Brenduns committed:
Final tweaks!

v-geberr committed:
new concept data manipulation

PRMerger16 committed:
Merge pull request #35413 from carolinacmoravia/patch-648

M24377:  Possible incorrect term: "IoT Hub and Device Provisioning Se…

rwike77 committed:
updated links

PRMerger17 committed:
Merge pull request #35336 from barbkess/wwi

updates

PRMerger17 committed:
Merge pull request #35426 from CarlRabeler/1TB

1 tb

j-martens committed:
AML: tutorial 2 updates to images

v-geberr committed:
edits

ktoliver committed:
Merge pull request #34224 from nitinme/nitinme-databricks-tutorial-eventshub

Adding a tutorial for Azure Databricks

ktoliver committed:
Merge pull request #34660 from bwren/la-overview

Log Analytics overview

v-geberr committed:
added link to regions for bot tutorials

v-geberr committed:
manipulation to alteration

Commit:
Merge pull request #35427 from Brenduns/vmupdates

Final tweaks!

dlepow committed:
wording change

Commit:
Merge pull request #35428 from curtand/admin0315

[Azure AD admin roles] remove preview from Information Protection

Commit:
Merge pull request #35433 from v-geberr/0315-regions-link-on-more-pages

added link to regions for bot tutorials

dlepow committed:
metadata

rwike77 committed:
acrolinx updates

PRMerger10 committed:
Merge pull request #35424 from mattbriggs/mdb_updatewebapp-api

adding web api clarification

rayne-wiselman committed:
Update vmware-azure-common-questions.md

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-adls-updates

v-geberr committed:
edits

Lisaco88 committed:
Merge pull request #35432 from j-martens/master

AML: tutorial 2 updates to images

itechedit committed:
Update custom-decision-service-log-downloader-reference.md

rwike77 committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into guestexes

PRMerger17 committed:
Merge pull request #35429 from dlepow/hpc

HPC VMs: RDMA clarification

ktoliver committed:
Merge pull request #35404 from rayne-wiselman/rayne-march15

new article, fixing issues 5757,5756,5738

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-adls-updates

Commit:
Content freshness work on Service Fabric

ShubhaVijayasarathy committed:
Adding output to the code

ktoliver committed:
Merge pull request #35299 from ggailey777/current

Clarify the VS tools update instructions

PRMerger20 committed:
Merge pull request #35416 from carolinacmoravia/patch-649

M24393: Typo mistake: "on" is missing between "information" and "the …

christiankuhtz committed:
tweak

christiankuhtz committed:
call out lb pip sku diffs

v-geberr committed:
bing speech priming and LUIS regions

Commit:
Merge pull request #35353 from ausfeldt/patch-19

update Table API differences FAQ

Commit:
automation: metadata cleanup

v-geberr committed:
add table for scanning

christiankuhtz committed:
updated date

Commit:
automation: corrected title

itechedit committed:
Update azure-stack-identity-architecture.md

PRMerger13 committed:
Merge pull request #35441 from georgewallace/autoinvupdates

automation: corrected title

Commit:
updates to meet acrolinx scores

PRMerger15 committed:
Merge pull request #35443 from v-geberr/0315-faq-speech-priming

bing speech priming and LUIS regions

carolinacmoravia committed:
L14222: Extra blanck space in link structure

Hello, @danielsollondon, 

Localization team has reported source content issue that causes localized version to have broken format compared to en-us version.  

<Provide further explanation here as needed>  

Please, help to check my proposed file change into the article and help to merge if you agree with fix. If not, please, let me know either if you would like me to fix it in another way within this PR, if you prefer to fix it in another PR or if I should close this PR as by-design. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR. 

Many thanks in advance.

PRMerger17 committed:
Merge pull request #35439 from christiankuhtz/patch-81

tweak

Commit:
corrected titles

itechedit committed:
Update azure-stack-identity-overview.md

ShawnJackson committed:
Merge pull request #34844 from itechedit/custom-decision-service-log-downloader-reference

edit pass: custom-decision-service-log-downloader-reference

GitHubber17 committed:
Merge pull request #35440 from christiankuhtz/patch-82

call out lb pip sku diffs

rwike77 committed:
reviewer comments

Lisaco88 committed:
Merge pull request #35431 from v-geberr/0315-concept-data-manipulation

new concept data manipulation

ShawnJackson committed:
Update azure-stack-identity-architecture.md

barbkess committed:
change develop-concurrency to resource-classes article

tfitzmac committed:
revised troubleshooting tips

GitHubber17 committed:
Merge pull request #35437 from ShubhaVijayasarathy/adding_sparkconnector

Adding output to the code

tfitzmac committed:
Style change

PRMerger10 committed:
Merge pull request #35146 from bwren/la-views

Log Analytics views - drill down

ShawnJackson committed:
Merge pull request #34935 from itechedit/azure-stack-identity-architecture

edit pass: azure-stack-identity-architecture

v-geberr committed:
concept prediction

GitHubber17 committed:
ms.date year value must be yyyy to show on staging

itechedit committed:
Update azure-stack-identity-overview.md

GitHubber17 committed:
Merge pull request #35425 from rwike77/guestexes

Guest exes refactoring

barbkess committed:
fix TOC problem

cjgronlund committed:
metadata fix only

BryanTrach-MSFT committed:
(forumsteam) fixes MicrosoftDocs/azure-docs#5639

Removed the leading \ from the two domains in the PowerShell script to create a self-signed cert. Reproduced that the leading \ causes an issue when using it for the app services. Creating certs without the leading \ worked for both the ILB cert or web app cert.

GitHubber17 committed:
Merge pull request #35454 from v-geberr/0315-concept-group-predictions

concept prediction

v-geberr committed:
concept enterprise strategies

v-geberr committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into 0315-concept-enterprise-customer

douglaslMS committed:
Added 2nd new Databricks article, updated TOC.

douglaslMS committed:
Fixed a typo.

v-geberr committed:
toc change

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-adls-updates

nitinme committed:
minor updates

nitinme committed:
more updates

douglaslMS committed:
Added a 2nd H2 before getting dinged for it. :-)

rjagiewich committed:
Merge pull request #35450 from tfitzmac/0314debug

revised troubleshooting tips

v-geberr committed:
edits

PRMerger13 committed:
Merge pull request #35458 from BryanTrach-MSFT/patch-4

(forumsteam) fixes MicrosoftDocs/azure-docs#5639

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

rjagiewich committed:
Merge pull request #35386 from JeffGoldner/master

Add Dell EMC CloudLink

v-geberr committed:
touch for rebuild

SnehaGunda committed:
Removing note and updating VS version

CarlRabeler committed:
[sql db] 1 TB

CarlRabeler committed:
more

anrothMSFT committed:
Merge pull request #1 from anrothMSFT/anrothMSFT-patch-1

add link to api reference

CarlRabeler committed:
Merge branch 'master' into 1TB2

v-alje committed:
Merge pull request #35466 from MicrosoftDocs/FromPublicRepo

From public repo

PRMerger20 committed:
Merge pull request #35455 from cjgronlund/ds-beg-fresh

metadata fix only - service correction

v-geberr committed:
concept - best practices

Commit:
Merge pull request #35464 from nitinme/nitinme-adls-updates

Minor updates

ktoliver committed:
Merge pull request #35460 from v-geberr/0315-concept-enterprise-customer

concept enterprise strategies

Commit:
Merge pull request #35470 from CarlRabeler/1TB2

1 tb2

v-geberr committed:
fix toc merge conflict

tfitzmac committed:
clarified validation delivery

Commit:
Merge pull request #35474 from linda33wj/master

Update ADF quickstart descriptions

SnehaGunda committed:
Removing note and updating VS version

tamram committed:
adding resources section to storage qs

v-geberr committed:
edits

anrothMSFT committed:
Merge branch 'master' of https://github.com/anrothMSFT/azure-docs-pr

anrothMSFT committed:
update manager

tfitzmac committed:
changed registration link

PRMerger15 committed:
Merge pull request #35477 from tfitzmac/0315validation

clarified validation delivery

v-geberr committed:
touch for rebuild

PRMerger17 committed:
Merge pull request #35480 from tfitzmac/0315publisher

changed registration link

PRMerger20 committed:
Merge pull request #35469 from SnehaGunda/master

Removing note and updating VS version

v-geberr committed:
acro score

Commit:
Updates to second article including screenshot

tamram committed:
fixing link and dates

ggailey777 committed:
Changed owner to Tom

ShawnJackson committed:
Merge pull request #35223 from itechedit/azure-stack-identity-overview

edit pass: azure-stack-identity-overview

JasonWHowell committed:
Update data-lake-analytics-quota-limits.md

curtand committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into linkedin0306

markgalioto committed:
adding include file to announce need to upgrade MARS agent

curtand committed:
last integration term edit

JasonWHowell committed:
Update data-lake-analytics-quota-limits.md

ggailey777 committed:
Fixed Tom's alias

tamram committed:
tweak verbiage

ktoliver committed:
Merge pull request #35475 from v-geberr/0315-best-practices

concept - best practices

mtillman committed:
added .NET core sample

JasonWHowell committed:
Update data-lake-analytics-quota-limits.md

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-databricks-sentiment-tutorial

# Conflicts:
#	articles/azure-databricks/TOC.yml

nitinme committed:
more updates

markgalioto committed:
fixing acrolinx scores

markgalioto committed:
fixing link in includes file

mtillman committed:
change name of app

curtand committed:
shorter page tite

Commit:
Merge pull request #35484 from JasonWHowell/patch-1

Update data-lake-analytics-quota-limits.md fixes MicrosoftDocs/azure-docs#5678

barbkess committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into load

barbkess committed:
fix acrolinx problems

muralikk committed:
Added the command line for bitlocker manage-bde -protectors -get D:

nitinme committed:
more updates

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-databricks-sentiment-tutorial

Commit:
gdpr01

barbkess committed:
acrolinx fixes

Commit:
gdpr01

Commit:
gdpr01

Commit:
gdpr01

PRMerger15 committed:
Merge pull request #35490 from muralikk/patch-22

Added the command line for bitlocker manage-bde -protectors -get D:

anrothMSFT committed:
Merge remote-tracking branch 'upstream/master'

MightyPen committed:
Add link to REST API.

MightyPen = GeneMi.

BrentL-Collabera committed:
Updated SaaS Azure Marketplace topic and TOC

Updated SaaS Azure Marketplace topic and TOC

mrbullwinkle committed:
add snapshot collection ref

Commit:
Merge pull request #35492 from anrothMSFT/master

add link to api reference

BrentL-Collabera committed:
Delete Saas Applications Technical Publishing Guide.md

This file had a filename that didn't follow the Azure Guidelines.

BrentL-Collabera committed:
Merge branch 'master' into New_Azure_Marketplace_Docs

PRMerger12 committed:
Merge pull request #35494 from mrbullwinkle/mrb_15_2018_core

add snapshot collection ref

BrentL-Collabera committed:
Update TOC.md

Fixed typo

rjagiewich committed:
Merge pull request #34640 from curtand/linkedin0306

[Azure AD admin] enable LinkedIn integration UI update

rjagiewich committed:
Merge pull request #35491 from MarkusVi/gdpr01

gdpr01

Commit:
Merge pull request #35478 from tamram/tamram-0313

adding resources section to storage qs

rachel-msft committed:
update mysql monitoring article

rjagiewich committed:
Merge pull request #35449 from barbkess/load

change develop-concurrency to resource-classes article

ajaycode committed:
do not use SCOM agents for ER-M

neilpeterson committed:
fixes MicrosoftDocs/azure-docs#5778

PRMerger16 committed:
Merge pull request #35295 from ajaycode/network-monitoring-overview

linked PM, SEPM and ERM to NPM

PRMerger17 committed:
Merge pull request #35483 from ggailey777/patch-14

Changed owner to Tom

markgalioto committed:
fixing the link to the includes file

neilpeterson committed:
update

MightyPen committed:
Clarify Next Step entry by appending REST APIs.

And update the YAML date.

markgalioto committed:
tweaked the include file a little

mimig1 committed:
Cosmos DB: Corrected typo per verbatim

PRMerger15 committed:
Merge pull request #35493 from MightyPen/patch-4

Add link to REST API.

nsoneji committed:
updaed the date

PRMerger12 committed:
Merge pull request #35501 from nsoneji/master

updaed the date

dominicbetts committed:
Reassign C articles to Yi

ajaycode committed:
TA: added capacity utilization

PRMerger18 committed:
Merge pull request #35507 from dominicbetts/reassign-c

Reassign C articles to Yi

PRMerger20 committed:
Merge pull request #35508 from ajaycode/network-monitoring-overview

TA: added capacity utilization

ajaycode committed:
minor edit

PRMerger13 committed:
Merge pull request #35500 from mimig1/patch-225

Cosmos DB: Corrected typo per verbatim

Commit:
Merge pull request #35445 from carolinacmoravia/patch-650

L14222: Extra blank space in link structure

v-geberr committed:
edits

Commit:
Merge pull request #35498 from neilpeterson/github-issue-5778

fixes MicrosoftDocs/azure-docs#5778

rmca14 committed:
Merge pull request #35438 from craigcaseyMSFT/v-craic0315

Content freshness work on Service Fabric

mtillman committed:
Moved to new section

alkohli committed:
Removed the reference that v 8 - 11 are supported 

Vijay reported that in the field when a v11 was booted, it didn't work.

GitHubber17 committed:
Merge pull request #35497 from rachel-msft/mysqlmon

update mysql monitoring article

PRMerger15 committed:
Merge pull request #35513 from v-geberr/0316-enterprise

edits

PRMerger16 committed:
Merge pull request #35488 from mtillman/mtillman315mstr

Issue 5202: add sample for .NET Core

carolinacmoravia committed:
Update tutorial-transform-data-hive-virtual-network-portal.md

gsacavdm committed:
Update gov extensions & imgs

CarlRabeler committed:
rl

CarlRabeler committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

CarlRabeler committed:
Update sql-database-servers-databases.md

Commit:
Merge pull request #32139 from carolinacmoravia/patch-557

M21532: Potential incorrect terms

CarlRabeler committed:
Update sql-database-managed-instance.md

MikeDodaro committed:
move Go sample down in TOC

BrentL-Collabera committed:
Update TOC.md

Changed headings per feedback

CarlRabeler committed:
australia

BrentL-Collabera committed:
Update TOC.md

Sentence case for top heading per Azure guidelines.

Commit:
Merge pull request #35514 from alkohli/patch-180

Removed the reference that v 8 - 11 are supported

Ja-Dunn committed:
Merge pull request #34945 from ShawnJackson/migrate-adfs-apps-to-azure

edit pass: migrate-adfs-apps-to-azure

PRMerger10 committed:
Merge pull request #35518 from CarlRabeler/patch-28

Update sql-database-servers-databases.md

PRMerger12 committed:
Merge pull request #35521 from CarlRabeler/patch-29

Update sql-database-managed-instance.md

PRMerger13 committed:
Merge pull request #35442 from georgewallace/autometadata

automation: metadata cleanup

GitHubber17 committed:
Merge pull request #35520 from MikeDodaro/moveGosample

move Go sample down in TOC

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

GitHubber17 committed:
Update transform-data-databricks-notebook.md

Commit:
automation : metadata cleanup #2

markgalioto committed:
adding metadata to the Include article per guidance

PRMerger17 committed:
Merge pull request #35510 from ajaycode/network-monitoring-overview

minor edit

rmca14 committed:
Merge pull request #35516 from gsacavdm/gov-img-ext

Update gov extensions & imgs

GitHubber17 committed:
Merge pull request #35462 from douglaslMS/adf-databricks-2

Added 2nd new Databricks article, updated TOC.

Commit:
updates

PRMerger19 committed:
Merge pull request #32024 from carolinacmoravia/patch-554

M21517: Clarification request:  "Select Public Network" vs. "Select P…

v-alje committed:
Merge pull request #35523 from MicrosoftDocs/FromPublicRepo

From public repo

Commit:
Merge pull request #35524 from georgewallace/autometadata

automation : metadata cleanup #2

Commit:
Merge pull request #35517 from CarlRabeler/master

rl

rmca14 committed:
Merge pull request #34839 from BrentL-Collabera/New_Azure_Marketplace_Docs

Adding New Topic for Publishing SaaS Applications in the Azure Marketplace

Lisaco88 committed:
Merge pull request #35485 from markgalioto/mar152018

adding include file to announce need to upgrade MARS agent

Lisaco88 committed:
Merge pull request #34753 from nitinme/nitinme-databricks-sentiment-tutorial

Databricks sentiment analysis tutorial
